tests/test_qdrant_client.py

```diff
@@ -1105,10 +1105,62 @@ def test_client_close():
         local_client_persist_2.close()
     # endregion local
 
+@pytest.fixture(autouse=True)
+def fastembed_setup(request):
+    if request.node.get_closest_marker('fastembed'):
+        os.system('pip install fastembed')
+    elif request.node.get_closest_marker('no_fastembed'):
+        os.system('pip uninstall -y fastembed')
```

Comment:
Not sure whether i would like to install dependencies inside python tests via `os.system`
Is not it better to delegate package installation to a bash script and run tests twice with a corresponding marker? 
Imho, might be more flexible during the development process and a bash script is a more obvious place to install dependencies

Reply:
No longer doing this